src/content/docs/en/guides/server-side-rendering.mdx

```diff
@@ -366,3 +366,33 @@ const factPromise = fetch('https://catfact.ninja/fact')
 ```
 
 In this example, `A name` will render while `personPromise` is loading. At that point, `A fact` will be appear, while `factPromise` is loading.
+
+
+#### Split the SSR build into multiple files
+
+By default, Astro builds all pages into one single file called `entry.mjs`. With the `split` option, you tell Astro emit
+multiple files, where the number of files is the number of pages:
```

Comment:
These also should be one line I think
```suggestion
By default, Astro builds all pages into one single file called `entry.mjs`. With the `split` option, you tell Astro to emit multiple files, where the number of files is the number of pages:
```

Reply:
The newline doesn't have any effect on the rendered markdown, I added a new line only to have the text in check without scrolling :)